architecture/consumption-architecture-dependency-management.md

```diff
@@ -0,0 +1,148 @@
+2. To provide an easy model for our end user organizations large and small to have a framework/reference architecture to help them think about adopting OpenSSF technical projects and guidance. 
+
+This document is based on the previous work conducted in OpenSSF [BEST Working Group](https://github.com/ossf/wg-best-practices-os-developers):
+* [Open Source Software Supply Chain Threats, Threats Definition](https://github.com/ossf/toolbelt/tree/main/threats). 
+* [The Security Toolbelt](https://github.com/ossf/Diagrammers-Society/blob/main/drawings/Sterling%20Toolchain%20Patterns.svg).
+
+This document serves as a map that takes readers from here to OpenSSF and adjacent technologies. Details of the technologies are out of the scope of this document. The details of how consumers manage open source software dependencies are out of the scope of this document. 
+
+## Intended Audience
+Open source software maintainers, contributors and consumers are the intended audience of this document.
+
+The document answers these questions:
+* As an open source software producer, I'd like to adopt open source security technologies to produce more secure open source software. What resources does OpenSSF provide? 
+* As an open source software consumer, I'd like to adopt open source security technologies to consume secure open source software. What resources OpenSSF provide? 
+* As an open source software producer and a consumer, I'd like to contribute to open source security technologies, which OpenSSF projects need contributors? 
+
+## Nomenclature
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+# Open Source Software Dependency Management 
+Open source software is the foundation of the digital world we live in. It powers critical infrastructure and impacts people's everyday lives. 
+
+[SonaType latest state of the software supply chain report](https://www.sonatype.com/state-of-the-software-supply-chain/Introduction) has show among the monitored OSS ecosystems:
+* An average 29% of year-over-year growth of OSS projects
+* Demands for open source ecosystems are still growing, measured by the number of download requests
+* The number of malicious packages tripled in 2023 compared to 2022. 
+* In September 2023, 245,032 malicious packages were discovered, 2x all previous years (2019-2022) combined. 
+  
+[Synopsys latest open source security and risk analysis report](https://www.synopsys.com/software-integrity/resources/analyst-reports/open-source-security-risk-analysis.html?utm_source=google&utm_medium=cpc&utm_term=open_source_security_tool&utm_campaign=G_S_OSSRA_BMM&cmp=ps-SIG-G_S_OSSRA_BMM&gad_source=1&gclid=CjwKCAjwko21BhAPEiwAwfaQCKkvVOJOS1uu4S7zF1p9uqhcdwrCNXNbMl7EcD5L0SIgS58-jUj_OxoC7FwQAvD_BwE#introMenu) has shown that among the surveyed comemrcial software:
```

Comment:
Please run spellcheck. E.g.: "comemrcial".

Reply:
Thank you @david-a-wheeler for taking the time. Fixed the typo.